packages/timeline/README.md

```diff
@@ -45,3 +45,47 @@ This package is part of the mono repository "@hpcc-js" (aka Visualization Framew
 * _[Tutorials](https://github.com/hpcc-systems/Visualization/wiki/Tutorials)_
 * _[Gallery](https://raw.githack.com/hpcc-systems/Visualization/trunk/demos/gallery/gallery.html)_ ([alt](https://rawgit.com/hpcc-systems/Visualization/trunk/demos/gallery/gallery.html))
 * _[Wiki](https://github.com/hpcc-systems/Visualization/wiki)_
+
+<ClientOnly>
+  <hpcc-vitepress style="width:100%;height:600px">
+    <div id="target" style="width:100%;height:600px">
```

Comment:
Does this one need `width:100%` on the target div?

Reply:
Not sure the timeline doesn't render due to the dgrid issue